src/lib.rs

```diff
@@ -164,7 +164,7 @@
 //! ```
 //!
 //! Alternatively, if you expect to only have component implementing the trait for each entity,
-//! you can use the filter [`One`](crate::One). This has significantly better performance than iterating
+//! you can use the filter [`One`](One). This has significantly better performance than iterating
```

Comment:
```suggestion
//! you can use the filter [`One`](crate::One). This has significantly better performance than iterating
```

Due to a limitation with the `cargo-rdme` tool, all doc links in the README must start with `crate::`.